UI/GardenDeskForm.Designer.cs

```diff
@@ -49,14 +49,15 @@ private void InitializeComponent()
             lblDashboard = new Label();
             splitterDashboard = new Splitter();
             pnlTicketsOverview = new Panel();
+            lblTicketsOverview = new Label();
```

Comment:
![image](https://github.com/user-attachments/assets/930321bb-72d0-4300-bce9-a41c9b3c76f2)
![image](https://github.com/user-attachments/assets/8ef52756-3b65-4baf-8cd5-171f4bc049b9)

1. Please, remove the ID column, Dan told us we don't have to display it.
2. We need to keep the same design for labels and margins. Can you make it look like the Ticket Overview?